treeherder/model/derived/jobs.py

```diff
@@ -326,6 +326,16 @@ def retrigger(self, requester, job):
         """
         self._job_action_event(job, 'retrigger', requester)
 
+    def backfill(self, requester, job):
+        """
+        Issue a retrigger to the given job
```

Comment:
I would rather use the same description you used for the backfill endpoint below.